controller/src/beerocks/master/tasks/client_steering_task.cpp

```diff
@@ -146,14 +147,25 @@ void client_steering_task::steer_sta()
         TASK_LOG(DEBUG) << "SLAVE " << sta_mac
                         << " has an active socket, sending BACKHAUL_ROAM_REQUEST";
         auto roam_request =
-            message_com::create_vs_message<beerocks_message::cACTION_CONTROL_BACKHAUL_ROAM_REQUEST>(
```

Comment:
You added the `#include` to `BACKHAUL_STEERING_REQUEST_MESSAGE`.
Question: is it possible to remove the `#include` for `cACTION_CONTROL_BACKHAUL_ROAM_REQUEST`
(or is it part of a large `.h` file with many messages)

Reply:
The message is on `beerocks_message_control.h` which is not included in this file specifically. Probably included from another file.